pkg/reconciler/autoscaling/kpa/scaler.go

```diff
@@ -64,6 +64,7 @@ const (
 )
 
 var probeOptions = []interface{}{
+	prober.WithHeader(network.UserAgentKey, network.ActivatorUserAgent),
```

Comment:
This should be the autoscaler (specifically our KPA), the "activator" below is what we want to reach.

Reply:
Renamed to "network.AutoscalingUserAgent"